packages/terminal/src/browser/terminal-frontend-contribution.ts

```diff
@@ -724,11 +933,43 @@ export class TerminalFrontendContribution implements FrontendApplicationContribu
 
     protected async openTerminal(options?: ApplicationShell.WidgetOptions): Promise<void> {
         const cwd = await this.selectTerminalCwd();
-        const termWidget = await this.newTerminal({ cwd });
-        termWidget.start();
+        let profile = this.profileService.defaultProfile;
+
+        if (!profile) {
+            throw new Error('There are not profiles registered');
+        }
+        if (profile instanceof ShellTerminalProfile) {
+            profile = profile.modify({ rootURI: cwd });
```

Comment:
@tsmaeder, I believe this should have just been `modify({cwd})`. `rootURI` is not a recognized field when these options are eventually passed into the TerminalWidgetFactory.